@here/harp-mapview/lib/MapView.ts

```diff
@@ -1187,6 +1187,17 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_zoomLevel;
     }
 
+    /**
+     * Returns the storage level for the given camera setup.
```

Comment:
Please add something like this: "Actual storage level of the rendered tile data also depends on [[Tile.storageLevelOffset]]."

Reply:
Will do.